### PR TITLE
[PDXD-100] Alterations to third-party G+ API wrapper

### PR DESCRIPTION
**Pull request by nonvirtualthunk**
- Ensures that InputStream is always closed by DefaultRequestHandler
- Updates versions of httpclient and commons-io, adds workaround to DefaultRequestHandler for bug introduced in newer httpclient version
- Makes the httpclient used in DefaultRequestHandler overrideable by subclasses
- Fixes situation where getPlaces(...) call would wait 3 second before returning even when only accessing 1 page of data (should only be necessary for multi-page)
- Adds circle.yml file for CI build
- Changes pom version to indicate womply specific branch

### CODE REVIEW

**Comment by akwirick** [comment]
+ (100 -  99)

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+machine:
+  java:
+    version: oraclejdk8
+dependencies:
+  pre:
+      # set maven settings.xml with artifactory authentication from environment variable
+    - echo $MVN_SETTINGS_XML > ~/.m2/settings.xml
+      # update maven version to be 2.1.2-womply-<branchname>-SNAPSHOT, that being the adjusted version
+      # of our forked version of the base library (originally just 2.1.2)
+    - mvn --batch-mode release:update-versions -DautoVersionSubmodules=true -DdevelopmentVersion=2.1.2-womply-${CIRCLE_BRANCH}-SNAPSHOT
```
Comment (jbarnettwomply) [inline circle.yml]: not sure if we want to do this or just hardcode to whatever is checked in?

Comment (nonvirtualthunk) [inline circle.yml]: I'm good with whatever, what would be the appropriate command to replace this with to accomplish that?

Comment (nonvirtualthunk) [inline circle.yml]: It is done

**Comment by jbarnettwomply** [comment]
+1

**Pull request closed without merging**